src/sidebar.js

```diff
@@ -94,13 +89,18 @@ class Sidebar extends React.Component {
     // filter out if a user starts swiping with a second finger
     if (!this.isTouching()) {
       let touch = ev.targetTouches[0];
+      let tcX = touch.clientX;
+      if (this.props.pullRight) {
+        tcX = this.state.sidebarWidth - tcX;
```

Comment:
Oh yeah to be honest I knew that but I was lazy, I'm sorry :/.

If I am correct we should edit the [touchSidebarWith](https://github.com/ababol/react-sidebar/blob/right-sidebar/src/sidebar.js#L192) method and do the logic here ?